tests/drop_counters/test_drop_counters.py

```diff
@@ -876,32 +875,45 @@ def test_non_routable_igmp_pkts(ptfadapter, duthost, setup, tx_dut_ports, pkt_fi
     # records=[gr_obj]).build()
     # The rest packets are build like "simple_igmp_packet" function from PTF testutils.py
 
-    ethernet_dst = {"membership_query": "01:00:5e:00:00:01",
-                    "membership_report": "01:00:5e:02:02:04",
-                    "leave_group": "01:00:5e:00:00:02"}
-    ip_dst = {"membership_query": "224.0.0.1",
-              "membership_report": "224.2.2.4",
-              "leave_group": "224.0.0.2"}
-    igmp_types = {"v1": {"membership_query": "\x11\x00\x0e\xfe\xe0\x00\x00\x01",
-                         "membership_report": "\x12\x00\x0b\xf9\xe0\x02\x02\x04"},
-                  "v2": {"membership_report": "\x16\x00\x07\xf9\xe0\x02\x02\x04",
-                         "leave_group": "\x17\x00\x08\xfd\xe0\x00\x00\x02"},
-                  "v3": {"membership_query": "\x11\x00L2\xe0\x00\x00\x01\x01}\x00\x02\xac\x10\x0b\x01\n\x00\x00;",
-                         "membership_report": "\"\x009\xa9\x00\x00\x00\x01\x01\x00\x00\x02\xe0\x02\x02\x04\xac\x10\x0b\x01\n\x00\x00;"}
-    }
+    from scapy.contrib.igmp import IGMP
+    Ether = testutils.scapy.Ether
+    IP = testutils.scapy.IP
```

Comment:
@yvolynets-mlnx can we use also import here?

Reply:
For now no.
Because as you can see IGMP is imported directly from scapy, but not from ptf.
Is is done because for now current PTF version does not support IGMP packet field, but available scapy version does support IGMP but still only for version v1 and v2.

So it also explains why packet parameters for IGMP v3 is composed manually like hex.